src/adapter/v16.ts

```
@@ -68,11 +69,87 @@
       zwj: zwj || [],
     };
   },
-  async emojis({ version, force }) {
+  async emojis(ctx) {
+    const unicodeNames = await this.unicodeNames!(ctx);
+    const { sequences, zwj } = await this.sequences!(ctx);
```

ESLint

[error] 'sequences' is assigned a value but never used. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)